packages/suite/src/support/messages.ts

```diff
@@ -3842,6 +3842,10 @@ export default defineMessages({
         id: 'TR_TOR_DISABLE_ONIONS_ONLY_NO_MORE_DESCRIPTION',
         defaultMessage: 'You can safely disable Tor now.',
     },
+    TR_UNAVAILABLE_COINJOIN_TOR_DISABLE_TOOLTIP: {
+        id: 'TR_TRANSACTIONS_SEARCH_TOOLTIP',
+        defaultMessage: 'Unavailable. Tor is disable.',
```

Comment:
Tor is disabled

Reply:
Fixed https://github.com/trezor/trezor-suite/pull/6796/commits/448c5d6f3c719b69af9347afb96607d625c9aef2